msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1146,7 +1147,7 @@ The `extensions.autoRunEvents` property defines event-based activation extension
 
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
-|`events`| Array |20 | ✔️ | Configures the event that cause actions in an Outlook Add-in to run automatically. For example, see [use smart alerts and the `OnMessageSend` and `OnAppointmentSend` events in your Outlook Add-ins](/office/dev/add-ins/outlook/smart-alerts-onmessagesend-walkthrough?tabs=jsonmanifest).|
+|`events`| Array |20 | ✔️ | Configures the event that cause actions in an Outlook add-in to run automatically. For example, see [use smart alerts and the `OnMessageSend` and `OnAppointmentSend` events in your Outlook add-ins](/office/dev/add-ins/outlook/smart-alerts-onmessagesend-walkthrough?tabs=jsonmanifest).|
```

Comment:
Looks like "Add-ins" should be capitalized?

Reply:
No, it doesn't. We only capitalize Office Add-ins or Office Add-in. The others are lowercase, e.g., Outlook add-ins, Excel add-ins.

Reply:
Ah, ok! Good to know.